delirium/dns/fakeresolver.py

```diff
@@ -1,6 +1,4 @@
 from dnslib.server import BaseResolver
-#from dnslib.dns import RCODE
-#from dnslib import QTYPE
 from dnslib import *
```

Comment:
non-internal `import *` statements are generally considered bad practice.  Was more than just `dns.RCODE` and `QTYPE` needed from `dnslib`?